src/EditorFeatures/ExternalAccess/Debugger/Microsoft.CodeAnalysis.ExternalAccess.Debugger.csproj

```diff
@@ -1,10 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.ExternalAccess.Debugger</RootNamespace>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net472</TargetFramework>
+    <UseWPF>true</UseWPF>
```

Comment:
I don't imagine this change was necessary? This reminds me of the other CodeLens change you weren't sure about.

Reply:
No.  this debugger lib *def* talks to EditorFEatures.  it is @tmat 's EA layer so that he can populate the FAR window with symbolic results.  He'd have to say why this is needed.  BUt it will def have to change like this.